vtexdocs/devportal
# Build/rapidoc magic blocks

#### What is the purpose of this pull request?

To upgrade the RapiDoc version in order to render magic blocks contained in the description attributes of a OpenAPI document.

#### What problem is this solving?

The absence of body content, which includes magic blocks, in the API reference pages.

#### How should this be manually tested?

Go to [the preview page](https://deploy-preview-100--elated-hoover-5c29bf.netlify.app/docs/api-reference/catalog) and check if the magic blocks are correctly rendered. Review comments should be added to the [corresponding RapiDoc pull request](https://github.com/vtexdocs/RapiDoc/pull/11).

#### Screenshots or example usage

[Further information and screenshots can be found here.](https://www.notion.so/vtexhandbook/Tratar-blocos-OpenAPI-com-Rapidoc-0f5a21023fbe48ce9ccc17a56d5824ed#b70ee7a657b54016bbb4aa78759ffad9)

#### Types of changes
